frontend_app_gallery/test/container/Gallery.spec.js

```diff
@@ -0,0 +1,147 @@
+import React from 'react'
+import { expect } from 'chai'
+import { shallow } from 'enzyme'
+import { Gallery } from '../../src/container/Gallery.jsx'
+import { APP_FEATURE_MODE } from 'tracim_frontend_lib'
+import pictures from '../fixture/content/pictures.js'
+import { debug } from '../../src/debug.js'
+import { defaultDebug } from 'tracim_frontend_lib'
+
+describe('<Gallery />', () => {
+  const folderId = 1
+  const folderName = 'folderTest'
+
+  const props = {
+    i18n: {},
+    registerCustomEventHandlerList: () => { },
+    registerLiveMessageHandlerList: () => { },
+    setApiUrl: () => { },
+    data: {
+      config: {
+        apiUrl: 'http://localhost:1337/api',
+        slug: 'gallery',
+        faIcon: 'paperclip',
+        hexcolor: '#ffa500',
+        label: 'Gallery',
+        appConfig: {
+          forceShowSidebar: false,
+          workspaceId: 0
+        },
+        history: {
+          location: {
+            search: `?folder_ids=${folderId}`
+          }
+        },
+        translation :''
+      },
+      loggedUser: {
+        ...defaultDebug.loggedUser
+      }
+    },
+    t: key => key
+  }
+
+  const stateMock = {
+    displayedPictureIndex: 1,
+    imagesPreviews: pictures.map(picture => ({
+      src: picture.filename,
+      label: picture.label,
+      contentId: picture.content_id,
+      fileName: picture.filename,
+      lightBoxUrlList: [picture.filename],
+      previewUrlForThumbnail: [picture.filename],
+      rotationAngle: 0,
+      rawFileUrl: [picture.filename]
+    }))
+  }
+
+  const wrapper = shallow(<Gallery {...props} />)
+
+  describe('TLM handlers', () => {
+    describe('handleContentDeleted', () => {
+      describe('delete a picture', () => {
+        it('should not be in the picture list anymore', () => {
+          wrapper.setState(stateMock)
+          wrapper.instance().handleContentDeleted({content: pictures[1]})
+          expect(wrapper.state().imagesPreviews.every(image => image.contentId !== pictures[1].content_id))
+          expect(wrapper.state().imagesPreviews.length).to.equal(stateMock.imagesPreviews.length - 1)
+        })
+
+        it('should go to the next picture', () => {
```

Comment:
add a describe per case

Reply:
I'm updating the descriptions so they are clearer, but I'm keeping the describe / it as is because I'm not so sure that this should changed after all